examples/helihost.nwk

```diff
@@ -0,0 +1 @@
+((((aglaope_EastPE,amaryllis_EastPE)m4,(ecuadoriensis_EastE,malleti_EastE)m5)m3,((thelxiopeia_EastFG,melpomene_EastFG)m7,melpomene_EastT)m6)m2,(((melpomene_WestPA,rosina_WestCR)m10,rosina_WestPA)m9,(melpomene_EastC,cythera_WestE)m11)m8)m1;
```

Comment:
Same as others, renamed to `heliconius_host.nwk`.

Reply:
Removed.